mslice/plotting/pyplot.py

```diff
@@ -2603,7 +2624,7 @@ def loglog(*args, **kwargs):
 
 # This function was autogenerated by boilerplate.py.  Do not edit as
 # changes will be lost
-@_autogen_docstring(Axes.magnitude_spectrum)
+#@_autogen_docstring(Axes.magnitude_spectrum)
```

Comment:
The decorator works fine. It's just that some of the properties don't exist in older versions of Matplotlib so it gives an error. I don't quite know how to disable the decorator for specific versions. I'll see if google or stack overflow comes up with something.